src/Common/Core/Impl/Microsoft.Common.Core.csproj

```diff
@@ -42,6 +42,7 @@
     <Reference Include="System.IO.Compression.FileSystem" />
     <Reference Include="System.Security" />
     <Reference Include="System.Windows.Forms" />
+    <Reference Include="WindowsBase" />
```

Comment:
`Common.Core` should have no references to `WindowsBase`. The whole purpose of `IMainThread` abstraction is to avoid referencing WPF assemblies in utility library.